doc/user-manual/tools/command-line-options.rst

```diff
@@ -732,6 +732,16 @@ Other features
      Enable typing rule ``Setω : Setω`` (this makes Agda inconsistent;
      see :ref:`omega-in-omega <omega-in-omega>`).
 
+.. option:: --level-universe
+
+     .. versionadded:: 2.6.4
+
+     Makes ``Level`` live in its own universe ``LevelUniverse`` and
+     disallows having levels depend on terms that are not levels themselves.
+     When this option is turned off, ``LevelUniverse`` still exists,
+     but reduces to ``Set``.
+     Note : This option is currently not compatible with cubical primitives
```

Comment:
ditto

Please link back to the section in `universe-levels.rst`, even though it now just contains a copy of this text.  It might be extended later, and it helps to keep these copies in sync.